src/main/java/db/Storage.java

```diff
@@ -0,0 +1,15 @@
+package db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class Storage {
+    public static final Map<String, Integer> fruits = new HashMap<>();
+
+    public static void putOrChange(String fruitName, int quantity) {
```

Comment:
here better to create fruit DAO
```suggestion
    public static void put(String fruitName, int quantity) {
```